nat-lab/tests/telio.py

```diff
@@ -534,6 +534,7 @@ async def run(
     ) -> AsyncIterator["Client"]:
         if isinstance(self._connection, DockerConnection):
             start_tcpdump(self._connection.container_name())
+        await self.clear_core_dumps()
```

Comment:
Maybe it should go to the call site where `start_tcpdump` is being called? This implies that `start_tcpdump` is the starting point of the container which doesn't _feel_ like it(might be wrong) :)

Reply:
You mean inside the if statement like this? 
```python
        if isinstance(self._connection, DockerConnection):
            start_tcpdump(self._connection.container_name())
            await self.clear_core_dumps()
```

Reply:
Hi, thanks for taking a look, I'm not sure I understand correctly what you mean, I tried to follow the pattern of `tcpdump()` or collecting logs.

Where each test case would have a core dump, if one was created during the duration of each individual test. Of course if the container itself crashes or maybe `libtelio_remote.py`, we won't be able to collect core dumps in that case, but the same goes for logs and pcaps. 

`clear_core_dumps()` only deletes the old core dumps at the beginning of the test, so that we know if any present, were created during the duration of the test case.

the core dumps themselves are collected in `collect_core_dumps()` here

```python
finally:
                    await self.save_logs()
                    await self.collect_core_dumps()
```